metrics/graphite.go

```diff
@@ -3,16 +3,22 @@ package metrics
 import (
 	"fmt"
 	"net"
+	"strings"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
-	graphite "github.com/cyberdelia/go-metrics-graphite"
+	"github.com/cyberdelia/go-metrics-graphite"
 	"github.com/kelseyhightower/envconfig"
-	metrics "github.com/rcrowley/go-metrics"
+	"github.com/pborman/uuid"
+	"github.com/rcrowley/go-metrics"
+
+	"github.com/allegro/mesos-executor/runenv"
 )
 
 const graphiteConfigEnvPrefix = "allegro_executor_graphite"
 
+var metricsUUID = uuid.New()
```

Comment:
This could be a local function variable.

Reply:
Nope, it is used in two functions.